doc.go

```diff
@@ -115,6 +115,8 @@ type Config struct {
 	RedirectionURL string `json:"redirection-url" yaml:"redirection-url"`
 	// RevocationEndpoint is the token revocation endpoint to revoke refresh tokens
 	RevocationEndpoint string `json:"revocation-url" yaml:"revocation-url"`
+	// skips the tls verification for openid provider communication
```

Comment:
Needs the variable name added to the comment ...as it's exported (go-vet will complain) .. i.e. 
// SkipOpenIDProviderTLSVerify skip ...